Sources/OktaOidc/iOS/Internal/OktaOidcSignOutHandlerIOS.swift

```diff
@@ -36,14 +36,12 @@ final class OktaOidcSignOutHandlerIOS: OktaOidcSignOutHandler {
         
         oidcClient.signOutOfOkta(authStateManager, from: presenter) { error in
             notFinishedOptions.remove(.signOutFromOkta)
+
             if error != nil {
                 failedOptions.insert(.signOutFromOkta)
             }
             
-            super.signOut(with: notFinishedOptions,
-                          failedOptions: failedOptions,
-                          progressHandler: progressHandler,
-                          completionHandler: completionHandler)
+            super.signOut(with: notFinishedOptions, failedOptions: failedOptions, progressHandler: progressHandler, completionHandler: completionHandler)
```

Comment:
Why has this been moved to one line? The multiline version of this call was easier to read.

Reply:
Resolved.